src/menelaus/datasets/__init__.py

```diff
@@ -0,0 +1,2 @@
+from menelaus.datasets.make_example_data import make_example_batch_data
```

Comment:
I think right now `tools` is excluded from the documentation. If we're going to swap it out for a `datasets` module, then we probably want to add a sentence saying what it's for as a docstring here.